distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/annotations/PaxExamRule.java

```diff
@@ -62,6 +64,12 @@ public PaxExamRule(Object testInstance) {
         this.testInstance = testInstance;
     }
 
+    public static void failWithThrowableStacktrace(Throwable t, String message) {
```

Comment:
Good point, didn't think of that. A new common logging class seems like a good way to go. Pretty sure we'll add new logging related methods in there before long :-)